test/functional/test_framework/util.py

```diff
@@ -326,6 +326,16 @@ def initialize_datadir(dirname, n, chain):
         os.makedirs(os.path.join(datadir, 'stdout'), exist_ok=True)
     return datadir
 
+def adjust_bitcoin_conf_for_pre_16_3(conf_file):
+    conf = open(conf_file,'r', encoding='utf8')
```

Comment:
python `with` context manager should simplify this

Reply:
Have changed it.